src/server/config/schema.js

```diff
@@ -197,6 +197,11 @@ export default () => Joi.object({
       }))
     }))
   }).default(),
+
+  savedObjects: Joi.object({
+    indexCheckTimeout: Joi.string().default('5s').regex(/^\d+(\.\d)?[sm]$/, 'number with either s (seconds) or m (minutes) suffix')
```

Comment:
It's pretty well established at this point that our configurable timeouts are number of seconds, so let's just keep up that trend.  This means slightly fewer moving parts that can break on us anyway.

Reply:
I assume you mean milliseconds?

Reply:
Yep!